setup.py

```diff
@@ -44,32 +41,39 @@ def package_data(pkg, root_list):
 
 BLOCKS = [
     'mentoring = mentoring:MentoringBlock',
-    'mentoring-dataexport = mentoring:MentoringDataExportBlock',
-]
 
-BLOCKS_CHILDREN = [
     'mentoring-table = mentoring:MentoringTableBlock',
-    'column = mentoring:MentoringTableColumnBlock',
-    'header = mentoring:MentoringTableColumnHeaderBlock',
+    'mentoring-column = mentoring:MentoringTableColumn',
     'answer = mentoring:AnswerBlock',
-    'quizz = mentoring:QuizzBlock',
-    'message = mentoring:MentoringMessageBlock',
-    'tip = mentoring:QuizzTipBlock',
-    'choice = mentoring:QuizzChoiceBlock',
-    'html = mentoring:HTMLBlock',
+    'answer-recap = mentoring:AnswerRecapBlock',
+    'mcq = mentoring:MCQBlock',
+    'rating = mentoring:RatingBlock',
+    'mrq = mentoring:MRQBlock',
+    'mentoring-message = mentoring:MentoringMessageBlock',
+    'tip = mentoring:TipBlock',
```

Comment:
@nedbat: Yeah, @antoviaque and I debated that issue. In the end we decided to try the more generic names in hopes that this repo can provide a few basic "building blocks" that other XBlocks can use, in the same way that the "mentoring" block is built by a composition of mcq/mrq/tip/etc. blocks. In other words, seeing the generic names as an opportunity to re-use code, more than a potential for conflict.

If it's going to be an issue though, it's better to change things now rather than later.